### PR TITLE
Move from Travis to github actions

### PR DESCRIPTION
**Pull request by marinaglancy**

### CODE REVIEW

File: .github/workflows/moodle-ci.yml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval 10s --health-timeout 5s --health-retries 3
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - php: '7.3'
+            moodle-branch: 'MOODLE_310_STABLE'
+            database: mariadb
```
Comment (mudrd8mz) [inline .github/workflows/moodle-ci.yml:33]: I don't think we need and should run the tests in MariaDB. AMOS is a single site installation and we know the environment it is and will be running at. Let's save the Earth resources by not wasting it on running tests that are not needed.

Comment (scara) [reply .github/workflows/moodle-ci.yml:33]: A past ref: https://github.com/moodlehq/moodle-local_amos/pull/8#issuecomment-560329105

HTH,
Matteo

Comment (marinaglancy) [reply .github/workflows/moodle-ci.yml:33]: sure! I will remove it, I was not sure on which db AMOS runs

Comment (marinaglancy) [reply .github/workflows/moodle-ci.yml:33]: (well apparently I was supposed to know from the PR8 but I forgot :) )

Comment (mudrd8mz) [reply .github/workflows/moodle-ci.yml:33]: Uff, and I am happy that my today's reply was the same like the one that @scara found. I hate it when I say something and someone links to an older comment where I was telling exact opposite!

**Comment by mudrd8mz** [comment]
Thank you @marinaglancy for your work in this. Inspired by your pull request and a few other existing examples of the workflow definition files, I baked some quick & dirty solution to start with.

**Pull request closed without merging**